### PR TITLE
adding two new testimonials

### PR DESCRIPTION
**Pull request by hectorkambow**
femi & shuaib

Adding two new student testimonials to testimonial section

### CODE REVIEW

**Comment by hectorkambow** [comment]
@hborrelli1 

Hey Harry, 


Could yo review this for me real quick - I think its all in place and it rendered fine on my local. 

But I rather get another page of eyes before I merge it. 

Just in case - these are 2 super new testimonials that came in today. 

Figured we could add them in the testimonial section 🚀

File: data/testimonialsData.yml
Comment (hborrelli1) [inline data/testimonialsData.yml:21]: There is a typo at the end of this line.

File: data/testimonialsData.yml
Comment (hborrelli1) [inline data/testimonialsData.yml:31]: Make sure there is an empty new line at the end of this file. Our linter configuration requires that.

**Review by hborrelli1 — APPROVED** [review]
Approved once the small tweaks are fixed,

**Comment by hectorkambow** [comment]
Thank you!

**Pull request merged**
